src/app/@ansyn/imagery-cesium/maps/cesium-map/cesium-map.ts

```diff
@@ -471,11 +471,11 @@ export class CesiumMap extends BaseImageryMap<any> {
 	}
 
 	zoomIn(): void {
-		this.mapObject.camera.zoomIn();
+		this.mapObject.camera.zoomIn(this.mapObject.camera.defaultZoomAmount * 0.5);
```

Comment:
maybe a config or a const would be better for the value of 0.5

Reply:
Good idea